nilearn/surface/tests/test_surface.py

```diff
@@ -1087,3 +1096,91 @@ def test_inmemorymesh_index_error(in_memory_mesh):
         IndexError, match="Use 0 for coordinates and 1 for faces"
     ):
         in_memory_mesh[2]
+
+
+def test_mean_img(surf_img_1d, surf_img_2d):
+    """Check that mean is properly computed over 'time points'."""
+    # one 'time point' image returns same
+    img = mean_img(surf_img_1d)
+
+    assert_surface_image_equal(img, surf_img_1d)
+
+    # image with left hemisphere
+    # where timepoint 1 has all values == 0
+    # and timepoint 2 == 1
+    two_time_points_img = surf_img_2d(2)
+    two_time_points_img.data.parts["left"][:, 0] = np.zeros(shape=4)
+    two_time_points_img.data.parts["left"][:, 1] = np.ones(shape=4)
+
+    img = mean_img(two_time_points_img)
+
+    assert_array_equal(img.data.parts["left"], np.ones(shape=(4,)) * 0.5)
+    assert img.shape == (img.mesh.n_vertices,)
+
+
+def test_get_min_max(surf_img_2d):
+    """Make sure we get the min and max across hemispheres."""
+    img = surf_img_2d()
+    img.data.parts["left"][:, 0] = np.zeros(shape=(4))
+    img.data.parts["left"][0][0] = 10
+    img.data.parts["right"][:, 0] = np.zeros(shape=(5))
+    img.data.parts["right"][0][0] = -3.5
+
+    vmin, vmax = get_min_max(img)
+
+    assert vmin == -3.5
+    assert vmax == 10
+
+
+def test_concat_imgs(surf_img_2d):
```

Comment:
I like short docstrings

Reply:
We should probably use ruff to start flagging those so you don't have to keep on reminding us.

At the moment the error for this D103 (I think) to make sure public functions have a doc string is ignored in the tests folders.

Reply:
If we do not ignored it, it flags around 900 functions with missing doc strings. This could be done over several PRs.

Am just a bit worried of adding meaningless doc strings.

```
def test_check_inputs_foo():
   """Test the check_inputs_foo function."""
```

Reply:
I see, in some cases the test name is transparent, but I find it helpful in general.